packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/ImageAndTextViewer.stories.js

```diff
@@ -1,17 +1,16 @@
 import { Box, Grommet } from 'grommet'
 import { Provider } from 'mobx-react'
 import React from 'react'
-import asyncStates from '@zooniverse/async-states'
 import zooTheme from '@zooniverse/grommet-theme'
 
 import { SubjectFactory } from '@test/factories'
 import mockStore from '@test/mockStore'
 
-import ImageAndTextViewerConnector from './ImageAndTextViewerConnector'
+import ImageAndTextViewer from '../ImageAndTextViewer'
```

Comment:
```suggestion
import ImageAndTextViewer from './'
```
Is this easier to read?

Reply:
Included with a few other cleanup edits.